src/components/TezosOperationView.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { getOperation } from '../Conseil';
+
+interface TezosOperationProps {
+    network: string;
+    id: string;
+}
+
+interface TezosOperationState {
+    data: object;
+}
+
+export class TezosOperationView extends React.Component<TezosOperationProps, TezosOperationState> {
+
+    constructor(props: TezosOperationProps) {
+        super(props);
+        this.state = {data: {}};
+        this.refreshData = this.refreshData.bind(this);
+        this.refreshData(props);
+    }
+
+    refreshData(props: TezosOperationProps) {
+        getOperation(this.props.network, this.props.id).
+        then(value => this.setState({data: JSON.parse(decodeURI(value))}));
```

Comment:
Heads up, `JSON.parse` and `JSON.stringify` can `throw`

Reply:
Made this a dedicated task as part of ticket #21.